packages/@aws-cdk/aws-redshift-alpha/lib/private/database-query-provider/table.ts

```diff
@@ -208,6 +209,12 @@ async function updateTable(
 
   await Promise.all(alterationStatements.map(statement => executeStatement(statement, tableAndClusterProps)));
 
+  const oldTableNamePrefix = oldResourceProperties.tableName.prefix;
+  if (tableNamePrefix !== oldTableNamePrefix) {
+    await executeStatement(`ALTER TABLE ${tableName} RENAME TO ${tableNamePrefix + tableNameSuffix}`, tableAndClusterProps);
+    return tableNamePrefix + tableNameSuffix;
```

Comment:
Again, it may improve readability to define a newTableName variable in place of the concatenation. Same nit, but here is a more concrete application.

Reply:
I don't think this would work, as the code would need to be refactored to use the `newTableName`, especially if I move this method call back to the top of the function

Reply:
done